config/prow/cluster/monitoring/mixins/lib/config.libsonnet

```
@@ -63,17 +63,17 @@ local config = {
 
   // Boskos endpoints to be monitored
   boskosResourcetypes: [
-    {instance: "104.197.27.114:9090", type: "aws-account", friendly: "AWS account"},
-    {instance: "104.197.27.114:9090", type: "gce-project", friendly: "GCE project"},
-    {instance: "35.225.208.117:9090", type: "gce-project", friendly: "GCE project (k8s-infra)"},
-    {instance: "104.197.27.114:9090", type: "gke-project", friendly: "GKE project"},
-    {instance: "104.197.27.114:9090", type: "gpu-project", friendly: "GPU project"},
-    {instance: "35.225.208.117:9090", type: "gpu-project", friendly: "GPU project (k8s-infra)"},
-    {instance: "104.197.27.114:9090", type: "ingress-project", friendly: "Ingress project"},
-    {instance: "104.197.27.114:9090", type: "node-e2e-project", friendly: "Node e2e project"},
-    {instance: "104.197.27.114:9090", type: "scalability-project", friendly: "Scalability project"},
-    {instance: "35.225.208.117:9090", type: "scalability-project", friendly: "Scalability project (k8s-infra)"},
-    {instance: "104.197.27.114:9090", type: "scalability-presubmit-project", friendly: "Scalability presubmit project"}
+    {job: "k8s-prow-builds-boskos", type: "aws-account", friendly: "AWS account"},
```

if you add `instance` back, the query I suggested above will preserve history.  Could add a TODO to drop in 6 months from current date, when all old data has aged out
```suggestion
    {instance: "104.197.27.114:9090", job: "k8s-prow-builds-boskos", type: "aws-account", friendly: "AWS account"},
```